.github/workflows/ci-low-cadence.yml

```diff
@@ -65,6 +65,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+        with:
```

Comment:
we do this? We don't want submodules.

Reply:
Because unzip the files are hard to maintain